originals/s.yaml

```
@@ -145,6 +145,20 @@
     - Puzzle
     - Arcade
 
+- name: 'Serious Sam: the Second Encounter'
+  external:
+    wikipedia: 'Serious Sam: The Second Encounter'
+  platform:
+  - DOS
```

```suggestion
  - Xbox
```
I don't think this was available for dos